runtime/near-vm-logic/tests/test_miscs.rs

```diff
@@ -559,51 +563,54 @@ fn test_ripemd160() {
     });
 }
 
+#[serde_as]
+#[derive(Deserialize)]
+struct EcrecoverTest {
+    #[serde_as(as = "Hex")]
+    m: [u8; 32],
+    v: u8,
+    #[serde_as(as = "Hex")]
+    sig: [u8; 64],
+    mc: bool,
+    #[serde_as(as = "Option<Hex>")]
+    res: Option<[u8; 64]>,
+}
+
 #[test]
 fn test_ecrecover() {
-    let mut logic_builder = VMLogicBuilder::default();
-    let mut logic = logic_builder.build(get_context(vec![], false));
+    for EcrecoverTest { m, v, sig, mc, res } in
+        from_slice::<'_, Vec<_>>(read(Path::new("tests/ecrecover-tests.json")).unwrap().as_slice())
```

Comment:
```suggestion
        from_slice::<'_, Vec<_>>(fs::read("tests/ecrecover-tests.json").unwrap().as_slice())
```

* no need to explicitly cast to Path
* it's idiomatic to qualify things from fs with fs